src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -684,31 +604,31 @@ class MatrixDelayedUpdateCUDA
     const int norb = psiMinv.rows();
     const int lda  = psiMinv.cols();
     const int nw   = engines.size();
-    resize_updateInv_scratch_arrays(nw);
+    updateInv_buffer_H2D.resize(sizeof(T*) * 6 * nw);
```

Comment:
Magic number remove or document again. This looks like it would benefit from cleanup later so maybe for here 
```c++
int num_updateInv_buffer_fields_per_engine = 6;
```

Reply:
Points taken.